krabsetw.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
     <metadata>
         <id>Microsoft.O365.Security.Krabsetw</id>
-        <version>4.2.2</version>
```

Comment:
Thanks for updating the nuspec! Would you be willing to update `O365.Security.Native.ETW.Debug.nuspec` and `O365.Security.Native.ETW.nuspec` as well?

Reply:
I think I've already done this?

https://github.com/microsoft/krabsetw/blob/e1d416ecac2a58fcaec531fda801a45d69c2c7f9/O365.Security.Native.ETW.Debug.nuspec#L4-L5
https://github.com/microsoft/krabsetw/blob/e1d416ecac2a58fcaec531fda801a45d69c2c7f9/O365.Security.Native.ETW.nuspec#L4-L5

Reply:
You're right, my apologies! I saw krabsetw.nuspec at the bottom of the changelist and didn't scroll up to find the others.